core/src/main/resources/db/cgds.sql

```diff
@@ -727,3 +729,10 @@ CREATE TABLE `info` (
 );
 -- THIS MUST BE KEPT IN SYNC WITH db.version PROPERTY IN pom.xml
 INSERT INTO info VALUES ('1.1.0');
+
+--1.2.0 specific changes:
```

Comment:
Yes, I thought about this as well. My reasoning for doing it like this was that I think it will just make it harder for people to correctly share their DB changes. I.e. it would mean one will have to register his changes in a set of `ALTER` statements (which he probably tested locally, so this is fine) and then think of how to translate them to the other `CREATE table` context syntax...which can be error prone and harder to test (i.e. to really test it one has to run the create statements in a separate DB).  

Let me know if you think this makes sense. I can translated this into the `CREATE table` parts if you still prefer that.